content/post/91-ipfs-dev-meetings-2018.md

```diff
@@ -0,0 +1,61 @@
+---
+date: 2019-06-03
+url: 91-ipfs-2018-dev-meetings
+tags: dev meetings ipfs libp2p community researchers
+title: July 2018 IPFS Developer Meetings
+author: TBD
+---
+![](https://ipfs.io/ipfs/QmQnUstaKTPA92XAkCj2cxBACX1nMyfjYeTHSawejGxASW)
+
+Last July we hosted the first ever IPFS and Libp2p developer meetings to bring together a small group of core developers and major users of the protocols. They were a huge success! We are incredibly thankful for everyone's participation, energy, enthusiasm - and for sharing their knowledge with the whole group.
+
+![](https://media.giphy.com/media/iCivRRnqS1SsadMjme/giphy.gif)
+
+The week was great for core developers, contributors and researchers to spend time together diving deep on open design problems, demoing exciting new tools and products being built on or around IPFS and Libp2p, and exploring the path forward for the protocols and working groups. Check out the [jam-packed schedule](https://developersmeetingsberlin2018.sched.com/) to see what we discussed.
+
+Much of the rich knowledge was shared in lightning talks and presentations that we’re excited to now share with the world! Learn about the Projects’ vision, get updated on what the projects were working on in mid 2018, and explore the ecosystem of demos, tools, apps, and exciting improvements to the core protocol in [this YouTube playlist](https://www.youtube.com/watch?v=ctjvfX1EoWY&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr). You can also see the full list below.
```

Comment:
On the note of pointing folks to the playlist, I wonder if organizing the talks list at https://github.com/ipfs/developer-meetings/ would make it more friendly to pick what to watch.

Perhaps we can c&p the list below to the repo as well?

Reply:
sure - took a pass: https://github.com/ipfs/developer-meetings/pull/48